### ISSUE TITLE
ScalaTask fails in script mode, but not in console

### ISSUE THREAD

**Issue by HKervadec** [openmole/openmole#78]
```scala
val i = Val[Double]
val res = Val[Double]

val exploration = ExplorationTask(i in (0.0 to 10.0 by 1.0))

val model = ScalaTask("val res = i * 2") set (inputs += i, outputs += (i, res))

val env = LocalEnvironment(4)

val ex = exploration -< (model on env hook ToStringHook()) start
// val ex = (exploration on env hook ToStringHook()) start
```

This script runs fine when pasted line by line into the console, but will fail at the last line when launched as a script (openmole -c -s toto).

(First 30 lines of the 400 lines error message)
```
SEVERE: Error in user job execution for capsule scalaTask0, job state is FAILED.
org.openmole.core.exception.InternalProcessingError: Error for context values in scalaTask0 {i=2.0, oMSeed=-3443810242302030532}
	at org.openmole.core.workflow.tools.InputOutputCheck$class.perform(InputOutputCheck.scala:94)
	at org.openmole.plugin.task.scala.ScalaTask.perform(ScalaTask.scala:33)
	at org.openmole.core.workflow.task.Task$class.perform(Task.scala:47)
	at org.openmole.plugin.task.scala.ScalaTask.perform(ScalaTask.scala:33)
	at org.openmole.core.workflow.job.MoleJob.perform(MoleJob.scala:99)
	at org.openmole.core.workflow.execution.local.LocalExecutor$$anonfun$1$$anonfun$apply$1.apply(LocalExecutor.scala:70)
	at org.openmole.core.workflow.execution.local.LocalExecutor$$anonfun$1$$anonfun$apply$1.apply(LocalExecutor.scala:62)
	at scala.collection.immutable.List.foreach(List.scala:381)
	at org.openmole.core.workflow.execution.local.LocalExecutor$$anonfun$1.apply(LocalExecutor.scala:62)
	at org.openmole.core.workflow.execution.local.LocalExecutor$$anonfun$1.apply(LocalExecutor.scala:59)
	at org.openmole.core.workflow.execution.local.LocalExecutor.withRedirectedOutput(LocalExecutor.scala:113)
	at org.openmole.core.workflow.execution.local.LocalExecutor.run(LocalExecutor.scala:59)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.openmole.core.exception.InternalProcessingError: Error compiling import org.openmole.core.workflow.tools.CodeTool._

(_input_value_context: org.openmole.core.workflow.data.Context, _input_value_RNGProvider: org.openmole.core.workflow.data.RandomProvider) => {
    object input {
      var i = _input_value_context("i").asInstanceOf[Double]; var oMSeed = _input_value_context("oMSeed").asInstanceOf[Long]; var workDir = _input_value_context("workDir").asInstanceOf[java.io.File]
    }
    import input._
    implicit lazy val oMRNG: util.Random = _input_value_RNGProvider()
    val res = i * 2
    
import scala.collection.JavaConversions.mapAsJavaMap
mapAsJavaMap(Map[String, Any](  "i" -> i, "res" -> res ))
    
}
```
On the other hand, the exploration works in both modes ( the commented line contain only an exploration task).

**Comment by HKervadec** [comment]
Adding ```ex.waitUntilEnded``` after the last line fix the problem.

Still, it is not quite straightforward, and the error message seems quite unrelated to the issue.

**Comment by jopasserat** [comment]
fixed by c5636090a48ab6c5f427f93755bec9ad653c75cc
